other/upgrade_2-1_postgresql.sql

```diff
@@ -1362,6 +1362,36 @@ DELETE FROM {$db_prefix}themes
 WHERE variable IN ('show_board_desc', 'no_new_reply_warning', 'display_quick_reply', 'show_mark_read', 'show_member_bar', 'linktree_link', 'show_bbc', 'additional_options_collapsable', 'subject_toggle', 'show_modify', 'show_profile_buttons', 'show_user_images', 'show_blurb', 'show_gender', 'hide_post_group', 'drafts_autosave_enabled', 'forum_width');
 ---#
 
+---# Update the SM Stat collection.
+---{
+	// First get the original value
+	$request = $smcFunc['db_query']('', '
+		SELECT value
+		FROM {db_prefix}settings
+		WHERE variable = {literal:allow_sm_stats}');
+	if ($smcFunc['db_num_rows']($request) > 0 && $row = $smcFunc['db_fetch_assoc']($request))
+	{
+		if (!empty($row['value']))
+		{
+			$smcFunc['db_insert']('',
```

Comment:
in my eyes you get no benefit,
because your beliefe "ther should be no value".
but why risking that instead of use replace and "you don't care what ever ther is".